airflow/providers/microsoft/azure/CHANGELOG.rst

```diff
@@ -38,6 +38,18 @@ Breaking changes
    ``WasbPrefixSensorTrigger``. These params will now need to passed through ``check_options`` param
 
 
+.. Review and move the new changes to one of the sections above:
+   * ``Revert "Provide the logger_name param in providers hooks in order to override the logger name (#36675)" (#37015)``
+   * ``add WasbPrefixSensorTrigger params breaking change to azure provider changelog (#36940)``
+   * ``Fix stacklevel in warnings.warn into the providers (#36831)``
+   * ``Fix WasbPrefixSensor arg inconsistency between sync and async mode (#36806)``
+   * ``Fix failed tasks are not detected in 'AzureBatchHook' (#36785)``
+   * ``Fix assignment of template field in '__init__' in 'container_instances.py' (#36529)``
+   * ``Standardize airflow build process and switch to Hatchling build backend (#36537)``
+   * ``Provide the logger_name param in providers hooks in order to override the logger name (#36675)``
+   * ``Prepare docs 1st wave of Providers January 2024 (#36640)``
+   * ``Speed up autocompletion of Breeze by simplifying provider state (#36499)``
```

Comment:
Azure provider was not ready to release in RC1 so the changelog is not ready.
I suggest to drop Azure from this wave. We are waiting for another PR which is also breaking change

Reply:
I am dropping it yep - see above list :)